packit_service/worker/jobs.py

```diff
@@ -736,11 +737,9 @@ def handle_pull_request(self):
             msg = f"There was an error while running the build: {ex}"
             r.report("failure", msg)
             return HandlerResults(success=False, details={"msg": msg})
-        timeout = 60 * 60 * 2
-        # TODO: document this and enforce int in config
```

Comment:
I expect a docs PR now :P

Reply:
on the other hand, the timeout will be obsolete once we start using fedmsg for completed copr builds, instead of polling

Reply:
hence I wouldn't document it